packages/cloud-functions/src/bigQuery.ts

```diff
@@ -0,0 +1,17 @@
+import { BigQuery } from '@google-cloud/bigquery'
+import { ENVIRONMENT } from './config'
+
+const projectId = 'celo-testnet-production'
+const dataset = 'mobile_wallet_production'
+const bigQuery = new BigQuery({ projectId: `${projectId}` })
+
+export function trackEvent(table: string, row: any) {
+  if (ENVIRONMENT?.toLowerCase() !== 'mainnet') {
```

Comment:
Nit: I think it may be slightly better to infer the environment from the GCP using the method outlined [here](https://stackoverflow.com/questions/44078037/how-to-get-firebase-project-name-or-id-from-cloud-function). Though it's definitely a low probability edge case, env variables can be edited and deleted outside of the deployment process so it's possible `config.env.name` gets changed